sematic/config/settings.py

```diff
@@ -461,7 +462,11 @@ def _apply_scopes_overrides(plugin_scopes: PluginScopes) -> None:
         if scope.value in os.environ:
             logger.debug("Overriding scope %s from environment variables", scope.value)
 
-            plugin_scopes[scope] = os.environ[scope.value].split(",")
+            plugin_scopes[scope] = [
+                plugin_path.strip()
```

Comment:
This is so that the env var for a given scope can have newlines and spaces in it, which is helpful for making it in helm so the plugins can be individually and conditionally enabled.

Reply:
hmm... newlines in env vars sounds very clunky

Reply:
Discussed offline. We aren't using newlines here; we are explicitly ignoring them. `foo.bar.Baz,qux.blah.Blarg` is still valid, we just ignore extra newlines/spaces. This is helpful so that we can have newlines in the scope in the helm configmap. In turn, that is helpful because it allows us to nicely construct the list by adding elements conditionally based on bools in the `values.yaml`.